src/Sparrow/Meta.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Sparrow
+{
+    public static class Meta
+    {
+        public interface IDescribe
```

Comment:
Why not just use `ToString`?

Reply:
I didn't finish the ``IDescribe`` interface yet, for now, it is just a marker. Allocators are composable so there are multiple metrics along the way and each allocator in the chain paints a different picture. We need to be able to gather information along the way in a more structured way. Also ``.ToString()`` is vulnerable to out of band calls via the debugger and inspectors, which may complicate things. I would move away from the ``.ToString()`` mechanism for general inspection altogether and keep it bare essential for identification purposes instead.

Reply:
I haven't killed ToString I don't want it to generate chained information from all allocators in the chain or use it to inspect data. The idea is that the describe will return structured information that can be translated to a json for inspection on higher levels

Reply:
The proper way to handle this is either: Return an _object_, not a string, that can be used for the debug endpoint.
Or: Use `ToString` and call it a day